### PR TITLE
New scout version 4.10

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added
- Rank score model in causatives page
- Exportable HPO terms from phenotypes page
- AMP guideline tiers for cancer variants
- Adds scroll for the transcript tab
- Added CLI option to query cases on time since case event was added
- Shadow clinical assessments also on research variants display
- Support for CRAM alignment files
- Improved str variants view : sorting by locus, grouped by allele.
- Delivery report PDF export
- New mosaicism tag option
- Add or modify individuals' age or tissue type from case page
- Display GC and allele depth in causatives table.
- Included primary reference transcript in general report
- Included partial causative variants in general report

### Fixed
- Fixed update OMIM command bug due to change in the header of the genemap2 file
- Removed Mosaic Tag from Cancer variants
- Fixes issue with unaligned table headers that comes with hidden Datatables
- Layout in general report PDF export
- Fixed issue on the case statistics view. The validation bars didn't show up when all institutes were selected. Now they do.
- Fixed missing path import by importing pathlib.Path
- Handle index inconsistencies in the update index functions
- Fixed layout problems

**Review:**
- [x] code approved by CR, DN, MM, MB, AR
- [x] tests executed by CR, DN, MM, MB, AR
- [x] "merge and deploy" approved by @ingkebil 

This is a minor version because it has bugfixes and new features. All the changes were previously reviewed, tested and approved.

### CODE REVIEW

**Comment by dnil** [comment]
☃️ 🎁

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [x.x.x]
```
Comment (dnil) [inline CHANGELOG.md]: It will come back and haunt you with the first new PR.. ;-)

Seriously, well done!

**Comment by northwestwitch** [comment]
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md]: Did I forget the date?

Comment (dnil) [reply CHANGELOG.md]: No, just as usual added the new changelog entry on version bump, and not immediately after. A reasonable optimisation that sometimes causes confusion on the first merge to the new development version.

Comment (northwestwitch) [reply CHANGELOG.md]: I wasn't confused about it! ;)

Comment (dnil) [reply CHANGELOG.md]: 😆 Of course not.

**Pull request merged**